ably-ios/ARTAuth.h

```diff
@@ -15,7 +15,7 @@
 
 @interface ARTAuthToken : NSObject
 
-@property (readonly, strong, nonatomic) NSString *id;
+@property (readonly, strong, nonatomic) NSString *idB64;
```

Comment:
If I read this correctly, this is the `token` string itself, so perhaps should be named accordingly.